hydra/test_utils/launcher_common_tests.py

```diff
@@ -388,13 +424,16 @@ def test_custom_task_name(
         task_config = OmegaConf.create(task_config or {})  # type: ignore
         cfg = OmegaConf.merge(task_launcher_cfg, task_config)
         assert isinstance(cfg, DictConfig)
+        output_tmp_dir, overrides = self.filter_tmp_dir(overrides)
```

Comment:
You are going through a lot of trouble to get a output temp dir, but the function is taking tmpdir as a parameter.
what is the difference?

Reply:
Same issue with the tmp dir is OS dependent. A temp dir that works on MAC OS doesnt work on Ubuntu :(